### PR TITLE
Turese/see matching

### PR DESCRIPTION
**Pull request by Turese**
Created ability to see matching for a project:

Makes a table to display events, people involved, and time on /projects/:project_id/matching.

If user does not own the project, then user cannot see project.
If project does not exist, then it doesn't crash.
If user not logged in, then it also doesn't crash.

### CODE REVIEW

File: features/see_matchings.feature [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Feature: See Rankings
+  
+
+
+
+    As an admin,
+    So that I can run the matching algorithm and see the results,
+    I want to see the ability to see matching results.
```
Comment (kev-chien) [inline features/see_matchings.feature]: minor English: I want the ability to see matching results.

File: app/controllers/matchings_controller.rb [hunk 2]
```diff
@@ -3,18 +3,26 @@ class MatchingsController < ApplicationController
...
+      @permission = current_user.id == @project.user.id
+    end
+    if @proj_exists and @matching
+      @is_matching = true
+      @parsed_matching = JSON.parse(@matching.output_json)
+    end
   end
 
-  # GET /project/:project_id/matching/new
+  # GET /projects/:project_id/matching/new
```
Comment (kev-chien) [inline app/controllers/matchings_controller.rb:16]: Thanks for fixing!

**Review by kev-chien — APPROVED** [review]
Looks good! I really appreciate how comprehensive the tests are, and how you handled multiple edge cases such as "project doesn't belong to user" and "project doesn't exist". Thanks!

**Review by lawrencechen14 — APPROVED** [review]
Looks good! Just a note about the security issues, your permissions checks do work for accessing matchings of other users, but they still don't cover accessing the projects pages of other users or accessing projects when logged out. This is fine though since all controllers will need to be modified to handle this security flaw, and I believe that this should be a story for someone to take on (albeit a shorter story).
